commands/trends_command.py

```diff
@@ -0,0 +1,219 @@
+import matplotlib
+import requests
+import pandas as pd
+from pytrends.request import TrendReq
+from datetime import date, timedelta
+
+from .botcommand import BotCommand
+from boterror import BotError
+from botquery import BotQuery
+from config import BASE_URL
+from constants import SEND_PHOTO_ENDPOINT, PHOTO_KEY, CHAT_ID_KEY, NEW_LINE_CHAR, SPACE, UNDERSCORE, PARSE_MODE_KEY, UNDERSCORE, SPACE
+
+"""
+returns the associated response dictionary key w.r.t to a country's name
+eg. United States -> united_states
+    canada -> Canada
+"""
```

Comment:
Missing capital on `returns`.
Also do you mean `Canada -> canada` here? because `united_states` is lowercase

Reply:
yes, good catch